docs/usage/getting-started.md

```diff
@@ -0,0 +1,83 @@
+# Getting started
+
+Tuist is a command line tool \(CLI\) that aims to facilitate the generation, maintenance, and interaction with Xcode projects. It's distributed as a binary, which you can easily install and use without having to depend on other tools to manage dependencies \(like you would do if the tool was written in other programming languages such as Ruby, or Java\).
+
+### Install
+
+The first thing that we need to do to get started is installing the tool. There are two recommended ways of doing it: using [Homebrew](https://brew.sh) or running a script. In either way, you need to run the following commands in your terminal:
+
+```bash
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/tuist/install/master/install)"
+```
+
+The process is relatively fast because we are actually not installing the tool. We are installing `tuistenv` \(which gets renamed to `tuist`\) when you install it.
+
+A very common issue working on iOS projects is **not having a reproducible environment**. Very often, projects depend on things that should be installed by other tools. To give you an example, if your project depends on [Fastlane](https://fastlane.tools/), chances are that it depends on [Bundler](https://bundler.io/) being installed in the system and a clean Ruby environment with the version that the project expects. If any of those things are missing or are not in a good state, it results in unexpected outputs and a really bad experience for your developers.  
+To avoid that, Tuist is self-contained and comes with `tuistenv` which ensures that the right version is used. It manages different versions in your environment and runs the version your project is pinned to. Thanks to that, we ensure that anyone in your team will use the same version of Tuist.  
+In a more advanced section on the documentation, we'll see the power of `tuistenv`. For now, we'll keep things simple and just assume that we are running Tuist directly.
+
+### Creating our first project
+
+Now that we have Tuist installed, we can create our first project. Create a directory for your app:
+
+```bash
+mkdir MyAppcd MyApp
+
+```
+
+And then run:
+
+```bash
+tuist init --platform ios --product application
+```
+
+The `init` command will bootstrap an iOS application, which includes the `Info.plist` files, an `AppDelegate.swift,` a tests file, and a **`Project.swift` that contains the definition of the project.**
+
+{% hint style="info" %}
+If you have used the Swift Package Manager before, the `Project.swift` file is the equivalent to the `Package.swift`.
+{% endhint %}
```

Comment:
This does not get rendered very well in markdown, maybe consider using a quote instead?

Reply:
Agree, that's a GitBook custom syntax.